datadog/main.tf

```diff
@@ -5,11 +5,15 @@ terraform {
   required_providers {
     datadog = {
       source  = "DataDog/datadog"
-      version = "2.13.0"
+      version = "~> 3.25"
```

Comment:
Could we pin this more strictly? That way, the demo will keep working, even if a future 3.x update makes breaking changes